test/tests/interfacekernels/1d_interface/reaction_1D_transient.i

```diff
@@ -0,0 +1,130 @@
+# Transient-state test for the InterfaceReaction kernel.
+#
+# Same to steady-state, except the following
+#
+# Natural BCs are applied (i.e. NewmannBC h=0 at left and right)
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 10
+  xmax = 2
+[]
+
+[MeshModifiers]
+  [subdomain1]
+    type = SubdomainBoundingBox
+    bottom_left = '1.0 0 0'
+    block_id = 1
+    top_right = '2.0 1.0 0'
+  []
+  [interface]
+    type = SideSetsBetweenSubdomains
+    depends_on = 'subdomain1'
+    master_block = '0'
+    paired_block = '1'
+    new_boundary = 'master0_interface'
+  []
+[]
+
+[Variables]
+  [u]
+    order = FIRST
+    family = LAGRANGE
+    block = '0'
+  []
+  [v]
+    order = FIRST
+    family = LAGRANGE
+    block = '1'
+  []
+[]
+
+[Kernels]
+  [diff_u]
+    type = MatDiffusion
+    D = '4'
```

Comment:
This should be `D_name`, not `D`. See https://www.mooseframework.org/source/kernels/MatDiffusion.html#matdiffusion

Reply:
Actually, since you've defined a Material with name D, you can just remove this line

Reply:
Actually, you can just remove this line.....`MatDiffusion` will default to a Material name of `D`, which you've defined.